src/unstract/adapters/x2text/llm_whisperer/src/llm_whisperer.py

```diff
@@ -71,26 +76,41 @@ def _make_request(
             "Authorization": f"Bearer {platform_service_api_key}",
         }
 
-        # Add files only if the request is for process
-        files = None
-        if "files" in kwargs:
-            files = kwargs["files"] if kwargs["files"] is not None else None
-
-        body = {
-            Constants.EXTRACTION_MODE: self.config.get(
-                Constants.EXTRACTION_MODE, ExtractionModes.TEXT.value
-            ),
-            Constants.OCR_MODE: self.config.get(
-                Constants.OCR_MODE, OCRModes.LINE_PRINTER.value
-            ),
-        }
+        # Add files only if the request is for whisper
+        data = None
+        params = None
+        if request_type == Constants.WHISPER and files is not None:
+            f = files["file"]
+            data = f.read()
+            headers["Content-Type"] = "application/octet-stream"
+
+            processing_mode_value = self.config.get(
+                Constants.PROCESSING_MODE, ProcessingModes.TEXT.value
+            )
+            if processing_mode_value == ProcessingModes.OCR.value:
+                processing_mode_value = (
+                    f"{processing_mode_value}?median_filter_size="
+                    f"{OCRFilters.MEDIAN_FILTER_SIZE.value}"
+                    f"&gaussian_blur_radius="
+                    f"{OCRFilters.GAUSSIAN_BLUR_RADIUS.value}"
+                )
+
+            params = {
+                Constants.PROCESSING_MODE: processing_mode_value,
+                Constants.OUTPUT_MODE: self.config.get(
+                    Constants.OUTPUT_MODE, OutputModes.LINE_PRINTER.value
+                ),
+            }
+
+        query_string = urlencode(params) if params else ""
+        llm_whisperer_svc_url += f"?{query_string}" if query_string else ""
 
         response = requests.post(
-            llm_whisperer_svc_url, headers=headers, files=files, data=body
+            llm_whisperer_svc_url, headers=headers, data=data
         )
         return response
 
-    def process(
+    def whisper(
```

Comment:
@athul-rs apparently this is needed only within LLMWhisperer and not the adapters itself. This would spell a lot of changes downstream and `whisper()` wouldn't make sense to other users/implementations